### ISSUE TITLE
[ENHANCEMENT] Keyboard Shortcuts

### ISSUE THREAD

**Issue by PsaltyDog** [danielgarthur/neanes#25]
Hello. First of all, I'd like to thank you very much for your program: I find it immensely helpful and now I do all my work with it. May God bless you!

I would find it even more helpful if there were a system of keyboard shortcuts for at least the most common neumes. I'm used to working mostly or solely through the keyboard and I find it quite a bit slower having to use the mouse.

If it would be helpful, I could try to compile a list of suggested shortcuts.

Thank you once more.
In Christ,
FF

**Comment by danielgarthur** [comment]
I did at one time experiment with keyboard entry and several different mappings but I couldn't decide on one that I liked. I'd like to make the mappings customizable by the user. If you post a suggested mapping, I can include it as a default mapping.

**Comment by PsaltyDog** [comment]
Yeah, it would make sense making them customisable; so many people have quite personal ways of using the keyboard.

I'm not really the right guy for this job, but I've given it a shot:

[neanes-win-keyboard-layout.pdf](https://github.com/danielgarthur/neanes/files/9894521/neanes-win-keyboard-layout.pdf)

Basically, I used the block of keys N-M-,-H-J-K-U-I-8 for the main ascending characters. Shift turns them into petaste; Ctrl turns them into descenders; Shift-Ctrl makes them descenders w/petaste. Alt is for kentimata, etc.

I've really tried to avoid any keybinding conflicts, but you never know. If you see anything wrong or something you would like to change, feel free; I'm not that attached. I've only done it for Windows, because that's all I have.

Thank you again for all your effort!

**Comment by danielgarthur** [comment]
@PsaltyDog I've added a lot of keyboard shortcuts. You can find the the shortcut list [here](https://danielgarthur.github.io/neanes/guide/keyboard.html). Please let me know what you think. I wanted to avoid using Control and Alt, so most of the "modifier" keys are single letters that can be typed with the left hand.

**Comment by PsaltyDog** [comment]
Nice. Thank you so much!

My comments:

1. Is there a reason you've doubled the shortcuts for +5 and +6?
2. It would be easier to remember if the shortcuts for 'ison+apostrophos' and 'apostrophos+apostrophos' used the same modifier key, since they are so visually similar.
3. A couple combinations have no shortcut: 'yporrhoi+kentimata' (would be R+D+M), 'synechesElaphron+kentimata' (would be R+F+M). You may have left them off because they're a bit awkward to type, but for me at least its not terribly difficult.
4. It would help me if there were also shortcuts for (a) 'send the martyria to the end of the line', (b) 'add a drop cap', and (c) for switching between the note insertion modes.

**Comment by danielgarthur** [comment]
@PsaltyDog 

1. Not intentional. I've removed the duplicates.
2. I agree. I've switched the modifiers to be the same.
3. Yes, that's why. I've added them, but they are accessed by holding Shift + R + M to get the running elafron, and R + / to get the yporroe (since it kind of looks like a slash).
4. I've expanded the martyria shortcuts (see the guide). I also added Ctrl + D to insert drop caps. Switching between input modes is now Ctrl + U and Ctrl + I to move left and right through the modes.
